packages/filesystem/src/browser/file-dialog/file-dialog-container.ts

```diff
@@ -31,13 +32,16 @@ export function createFileDialogContainer(parent: interfaces.Container): Contain
     child.unbind(FileTreeWidget);
     child.bind(FileDialogWidget).toSelf();
 
+    child.bind(FileDialogTree).toSelf();
+    child.rebind(Tree).toDynamicValue(ctx => ctx.container.get(FileDialogTree));
+
     child.bind(FileDialog).toSelf();
 
     return child;
 }
 
-export function createFileDialog(parent: interfaces.Container, props: FileDialogProps): FileDialog {
-    const container = createFileDialogContainer(parent);
+export function createFileDialog(parent: interfaces.Container, props: FileDialogProps, treeProps?: TreeProps): FileDialog {
+    const container = createFileDialogContainer(parent, treeProps);
```

Comment:
`TreeProps` are supposed to be rebound, please revert and rebind them here:
```ts
    const container = createFileDialogContainer(parent);
    container.rebind(TreeProps).toConstantValue({
        ...defaultTreeProps,
        multiSelect: props.canSelectMany
    });
```

Reply:
Done. Thanks!